Formula/t-rec.rb

```diff
@@ -0,0 +1,26 @@
+class TRec < Formula
+  desc "Blazingly fast terminal recorder that generates animated gif images for the web"
+  homepage "https://github.com/sassman/t-rec-rs"
+  url "https://github.com/sassman/t-rec-rs/archive/v0.2.1.tar.gz"
+  sha256 "a23ae0e19b76740220f4640a82d2813fe1002609bbf28ab180f3fc9228b141e3"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
+  depends_on "imagemagick"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    # let's fetch the window id
+    o = `#{bin}/t-rec -l | tail -1`
+    win_id = o.split(/\s|\n/)[-1]
+    # verify that it's an appropriate id
+    assert_equal win_id && Integer(win_id).positive?, true
```

Comment:
This isn't needed as it doesn't test much.

Reply:
Trust me on this, line 17 tests a lot of internal logic. I'd recommend leaving it in.

Reply:
In that case just do `shell_output("#{bin}/t-rec -l")`

Reply:
done